server/session/fork.go

```
@@ -131,6 +163,43 @@ func streamReplayIntoChild(parent *Store, child *Store, targetSeq int64) (replay
 	return derived, cutOrdinal, nil
 }
 
+func rebaseHistoryReplacementRollbackCandidate(
+	payload json.RawMessage,
+	locator *rollbacktarget.CandidateLocator,
+) (json.RawMessage, error) {
+	var engine historyReplacementEngine
+	if err := json.Unmarshal(payload, &engine); err != nil {
+		return append(json.RawMessage(nil), payload...), nil
```

golangci-lint (2.12.2)

[error] error is not nil (line 171) but it returns nil

(nilerr)